src/visualization/types.jl

```diff
@@ -529,6 +529,29 @@ function PlotData1D(u, mesh::TreeMesh, equations, solver, cache;
   if ndims(mesh) == 1
     x, data, mesh_vertices_x = get_data_1d(original_nodes, unstructured_data, nvisnodes)
     orientation_x = 1
+
+    # Special care is required for first-order FV approximations since the nodes are the
+    # cell centers and do not contain the boundaries
+    n_nodes = size(unstructured_data, 1)
+    if n_nodes == 1
+      n_visnodes = length(x) ÷ nelements(solver, cache)
+      if n_visnodes != 2
+        throw(ArgumentError("This number of visualization nodes is currently not supported for finite volume approximations."))
+      end
+      left_boundary = mesh.tree.center_level_0[1] - mesh.tree.length_level_0 / 2
+      dx_2 = zero(left_boundary)
+      for i in 1:(length(x) ÷ 2)
```

Comment:
```suggestion
      for i in 1:div(length(x), 2)
```
[Executable code should only use ASCII characters](https://trixi-framework.github.io/Trixi.jl/stable/styleguide/)

Reply:
Thanks for your correction.